frontend/public/components/pod.tsx

```diff
@@ -837,9 +838,9 @@ export const PodList: React.FC<PodListProps> = withUserSettingsCompatibility<
 });
 PodList.displayName = 'PodList';
 
-export const filters = [
+export const filters = (t: TFunction) => [
```

Comment:
Avoid passing the `t` function around since they're not all the same depending on how they were initialized. It's better to just use the global i18next instance.